packages/eas-cli/src/build/metadata.ts

```diff
@@ -20,15 +21,18 @@ export async function collectMetadata<T extends Platform>(
   }
 ): Promise<BuildMetadata> {
   return {
+    trackingContext: ctx.trackingCtx,
     appVersion: ctx.commandCtx.exp.version!,
     cliVersion: packageJSON.version,
     workflow: ctx.buildProfile.workflow,
     credentialsSource,
     sdkVersion: ctx.commandCtx.exp.sdkVersion,
-    trackingContext: ctx.trackingCtx,
+    releaseChannel: ctx.buildProfile.releaseChannel,
```

Comment:
what about a release channel that is not defined in eas.json?
I'm not saying to add that, just to make sure this is intended behaviour